pyaerocom/data/variables.ini

```diff
@@ -1939,7 +1964,8 @@ var_name = drypm10
 description = Dry deposition of PM10
 standard_name = tendency_of_atmosphere_mass_content_of_pm10_dry_aerosol_particles_due_to_dry_deposition
 var_type = dry deposition flux
-unit = kg m-2 s-1
+# unit = kg m-2 s-1
```

Comment:
dead code

Reply:
that comment I would like to keep because it's the unit according to the CF convention. Added quick comment about it.